.circleci/deployment/jobs.yml

```diff
@@ -151,3 +151,12 @@
           cf-password: CF_PASSWORD_PROD
           cf-space: tanf-prod
           cf-username: CF_USERNAME_PROD
+  deploy-clamav:
```

Comment:
The name conflict with the command is confusing. Can we rename this? Also, to match the rest of the file could you insert an empty line between this and the above job `deploy-production`?